FreeRDP/FreeRDP
# Flatpak support

* [x] Initial flatpak manifest, builds freerdp with dependencies
* [x] Default command is `xfreerdp`, but `wlfreerdp`, `freerdp-shadow-cli`, `sfreerdp-server`, `winpr-hash` and `winpr-makecert` can be run with `flatpak run --command=<cmd> com.freerdp.FreeRDP`
* [x] `xdg-download` folder is automatically shared for `/drive` redirection, additional folders must be excluded manually from the sandbox

TODO:
* [ ] Need to get feedback for missing permissions
* [ ] `libsystemd` dependency for `journald` logging currently deactivated
* [ ] Need to clean unused files from the flatpak
* [ ] Add to nightly builder
* [ ] Allow running X11 applications in wayland sessions with Xwayland

FUTURE:
* [ ] Submit to flathub.org

How to test:
* Build `flatpak-builder --repo=repo --force-clean flatpak/ packaging/flatpak/com.freerdp.FreeRDP.json`
  * Add a local repo `flatpak --user remote-add --no-gpg-verify tutorial-repo repo`
  * Install `flatpak --user install tutorial-repo com.freerdp.FreeRDP`
  * To update after changes `flatpak --user update`
* Run `flatpak run com.freerdp.FreeRDP` (should start the default `freerdp.sh` detection script launching `xfreerdp` or `wlfreerdp`)
